cpp/src/comms/cuML_comms_test.cpp

```diff
@@ -84,77 +81,31 @@ bool test_pointToPoint_simple_send_recv(const ML::cumlHandle& h,
     }
 
     communicator.waitall(requests.size(), requests.data());
-
-    if (0 == rank) {
-      std::cout << "There are " << communicator.getSize()
-                << " ranks:" << std::endl;
-    }
     communicator.barrier();
 
-    bool ret = true;
-    for (int rec : received_data) {
-      std::cout << "Rank received: " << rec << std::endl;
-      if (rec == -1) return false;
+    if (communicator.getRank() == 0) {
+      std::cout << "=========================" << std::endl;
+      std::cout << "Trial " << i << std::endl;
     }
```

Comment:
I think to order output there is a `communicator.barrier();` missing in line 90.

Reply:
Good idea. I was only concerned with making sure the trial number printed first. I can order the outputs as well.